autoload.php

```diff
@@ -46,7 +46,11 @@
     if (!empty($configFile)) {
         $neonConfig = Neon::decode(file_get_contents($configFile));
         if(is_array($neonConfig) && isset($neonConfig['parameters']) && isset($neonConfig['parameters']['tmpDir'])) {
-            $tmpDir = $neonConfig['parameters']['tmpDir'];
```

Comment:
We should add a call to \Nette\DI\Helpers::expand() in the same way as PHPstan does it to make sure we are dealing with the same configuration: https://github.com/phpstan/phpstan-src/blob/master/src/Command/CommandHelper.php#L149

Reply:
I think in that case, we'll need to add `nette/di` as a dependency (probably not a big deal). The class `Nette/DI/Helpers` is marked `@internal`, which my IDE treats like sorta like a deprecated class.

Given those points, are you comfortable using that method? I'd like to follow your lead on decisions like this.